packages/pixel-demo/test/swingsetTests/gallery/test-gallery.js

```diff
@@ -24,13 +24,13 @@ const expectedTapFaucetLog = [
   'pixel from faucet balance {"label":{"assay":{},"allegedName":"pixels"},"extent":[{"x":1,"y":4}]}',
 ];
 
-test('run gallery demo tapFaucet with SES', async t => {
+test.skip('run gallery demo tapFaucet with SES', async t => {
```

Comment:
Why keep these around? I guess I think of `test.skip` as a temporary bandage while something is known to fail, that will eventually graduate into a normal `test` (so we can write the tests first, then implement the feature). Is this `test.skip` because we think we might bring back this functionality?

Reply:
Yeah, I'm pretty sure we want to bring this example back, just after Zoe Alpha. So I think it fits the test.skip temporary bandage concept - it fails now, we know why, we know we need to redesign the feature, but we can't do it right now.